test/python/primitives/containers/test_bindings_array.py

```diff
@@ -431,3 +431,20 @@ def test_simple_vals(self):
             self.assertEqual(ba.shape, (1,))
             self.assertEqual(len(ba.vals), 1)
             np.testing.assert_allclose(ba.vals[0], [[1, 2, 3]])
+
+    def test_coerce(self):
```

Comment:
BindingsArrayLike is defined as follows. This test contains `BindingsArray`, `Mapping[Parameter, ArrayLike]`, and `None`. I suggest adding cases of `ArrayLike` and `Sequence[ArrayLike]` as well.
```python
BindingsArrayLike = Union[
    BindingsArray,
    ArrayLike,
    "Mapping[Parameter, ArrayLike]",
    "Sequence[ArrayLike]",
    None,
]
```

Reply:
`ArrayLike` and `Sequence[ArrayLike]` seem to be removed #11642